3Dmol/glmodel.js

```diff
@@ -2447,34 +2457,40 @@ $3Dmol.GLModel = (function() {
 
     /**
     * Set coordinates for the atoms parsed from various topology files. 
-    * @function $3Dmol.GLModel#setCoordinatesFromUrl
+    * @function $3Dmol.GLModel#setupFrames
     * @param {string} url - contains the url where mdsrv has been hosted
     * @param {string} path - contains the path of the file (<root>/filename)
-    * @param {object} viewer - contains the glviewer object
     */
 
-        this.setCoordinatesFromUrl = function(url, path) {
+        this.setupFrames = function (url, path) {
             var atomCount = atoms.length;
             var self = this;
             frames = [];
             $.get("http://"+url+"/traj/numframes/"+path, function (numFrames) {
```

Comment:
This is asynchronous, so if the next call (after setCoordinatesURL) is to setFrame, it won't work.